src/resolver/package_json.zig

```diff
@@ -1035,6 +1036,35 @@ pub const PackageJSON = struct {
                     }
                 }
             }
+            read_config: {
+                if (json.asProperty("config")) |config_prop| {
```

Comment:
this looks identical to the block above, might be a handy helper something like `config_prop.e_object.createStringMap()`